src/wollok/lang.wlk

```diff
@@ -638,6 +648,7 @@ class Collection {
    *      [1, 3, 5].findOrElse({ number => number.even() }, { 6.max(4) }) => Answers 6
    *      [].findOrElse({ number => number.even() }, { false })           => Answers false
    */
+  // TODO: Does this really need to be a native?
```

Comment:
We could give a try and port this back to Wollok language. Is it too painful to develop a `ts` solution? We could do performance tests with lists and sets to confirm the need for a native implementation

Reply:
It is already implemented in TS, but I feel like it's a bad idea to make Collection methods native, since it's an abstract class. I think it would be better to provide good default implementations as possible and, if needed for performance, just override with a native on the List/Set class.

That way new, custom collections (if ever one appears) can use the collection methods, and implementations of List and Set that are not too slow can delegate on super and save effort.